pages/app/index.tsx

```diff
@@ -77,10 +83,15 @@ function App() {
 }
 
 const history = createHashHistory();
-const { direction, visualRefresh } = parseQuery(history.location.search);
+const { direction, visualRefresh, removeHighContrastHeader } = parseQuery(history.location.search);
 
 // The VR class needs to be set before any React rendering occurs.
 window[awsuiVisualRefreshFlag] = () => visualRefresh;
+if (removeHighContrastHeader) {
+  window[awsuiGlobalFlagsSymbol] = {
```

Comment:
I'd recommend that we only set the individual flag of the object, currently with this code we'd override any existing flag object

Reply:
Addressed